src/sqs/index.ts

```diff
@@ -422,15 +423,15 @@ export class SqsQueue<MessagePayload> {
 			return;
 		}
 
-		if(!Number.isInteger(timeout) || timeout < 0) {
```

Comment:
Having all decimal numbers silently ignored here caused some problems. 

All our rate limiting timeouts were passing decimal numbers here (Due to Date.now() / 1000 ) 

We had to either throw an exception to make it more type-safe or support decimal numbers.

I've chosen to support decimal numbers and convert to the nearest integer for the following reasons:
1. Increase type-safety. If decimal passed we still execute logic
2. On the scale of visibility timeout it doesn't matter if we delay for 40.3 seconds or 40 seconds. So rounding the number doesn't break the desired logic